risc0/zkp/src/lib.rs

```diff
@@ -33,7 +33,7 @@ pub mod verify;
 
 pub use risc0_core::field;
 
-pub const MIN_CYCLES_PO2: usize = 13;
+pub const MIN_CYCLES_PO2: usize = 14;
```

Comment:
This is the zkp crate, and so this isn't specific to a particular circuit. Other circuits may indeed be smaller, so this value represents the limit of the proof system.

Reply:
I've moved it to `risc0-circuit-rv32im`